<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QmitkTensorReconstructionViewControls</class>
 <widget class="QWidget" name="QmitkTensorReconstructionViewControls">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>380</width>
    <height>1019</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>QmitkTensorReconstructionViewControls</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_11">
   <item>
    <widget class="QGroupBox" name="m_InputData">
     <property name="title">
      <string>Please Select Input Data:</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="1">
       <widget class="QLabel" name="m_DiffusionImageLabel">
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ff0000;&quot;&gt;mandatory&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_3">
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string>Raw DWI/DTI:</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Tensor reconstruction and colormap visualization</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QCheckBox" name="m_Advanced1">
        <property name="text">
         <string>Advanced Settings</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout_4">
         <property name="leftMargin">
          <number>9</number>
         </property>
         <property name="topMargin">
          <number>9</number>
         </property>
         <property name="rightMargin">
          <number>9</number>
         </property>
         <property name="bottomMargin">
          <number>9</number>
         </property>
         <item row="1" column="0">
          <widget class="QFrame" name="frame_3">
           <property name="frameShape">
            <enum>QFrame::NoFrame</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QGridLayout" name="gridLayout_3">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item row="1" column="0">
             <widget class="QLabel" name="m_TensorReconstructionThresholdLabel_2">
              <property name="text">
               <string>B0 Threshold</string>
              </property>
              <property name="wordWrap">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QSpinBox" name="m_TensorReconstructionThreshold">
              <property name="maximum">
               <number>10000</number>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QCheckBox" name="m_CheckNegativeEigenvalues">
           <property name="toolTip">
            <string>Only influences WLS reconstruction</string>
           </property>
           <property name="text">
            <string>Ignore voxels with negative eigenvalues</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QComboBox" name="m_ReconctructionMethodBox">
           <property name="currentIndex">
            <number>0</number>
           </property>
           <item>
            <property name="text">
             <string>Weighted Linear Least Squares</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>With correction for negative eigenvalues</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QCommandLinkButton" name="m_StartReconstruction">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="toolTip">
         <string>Select raw DWI!</string>
        </property>
        <property name="statusTip">
         <string/>
        </property>
        <property name="whatsThis">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>Start Reconstruction</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Estimate Diffusion Weighted Image from Tensors</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QFrame" name="m_TensorsToDWIFrame">
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QFormLayout" name="formLayout_3">
         <property name="fieldGrowthPolicy">
          <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
         </property>
         <property name="horizontalSpacing">
          <number>6</number>
         </property>
         <property name="verticalSpacing">
          <number>6</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="m_TensorsToDWIBValueLabel">
           <property name="toolTip">
            <string comment="how fuzzy the confidence boundary should be. By default, confidence boundary is perfectly sharp (float); default: &quot;0&quot;" extracomment="how fuzzy the confidence boundary should be. By default, confidence boundary is perfectly sharp (float); default: &quot;0&quot;">how fuzzy the confidence boundary should be. By default, confidence boundary is perfectly sharp (float); default: &quot;0&quot;</string>
           </property>
           <property name="statusTip">
            <string comment="how fuzzy the confidence boundary should be. By default, confidence boundary is perfectly sharp (float); default: &quot;0&quot;" extracomment="how fuzzy the confidence boundary should be. By default, confidence boundary is perfectly sharp (float); default: &quot;0&quot;">how fuzzy the confidence boundary should be. By default, confidence boundary is perfectly sharp (float); default: &quot;0&quot;</string>
           </property>
           <property name="whatsThis">
            <string comment="how fuzzy the confidence boundary should be. By default, confidence boundary is perfectly sharp (float); default: &quot;0&quot;" extracomment="how fuzzy the confidence boundary should be. By default, confidence boundary is perfectly sharp (float); default: &quot;0&quot;">how fuzzy the confidence boundary should be. By default, confidence boundary is perfectly sharp (float); default: &quot;0&quot;</string>
           </property>
           <property name="text">
            <string>B-Value</string>
           </property>
           <property name="wordWrap">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="m_TensorsToDWINumDirsLabel">
           <property name="text">
            <string>#Gradient Directions</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QComboBox" name="m_TensorsToDWINumDirsSelect">
           <property name="currentIndex">
            <number>3</number>
           </property>
           <item>
            <property name="text">
             <string>12</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>42</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>92</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>162</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>252</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>362</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>492</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>642</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>812</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>1002</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QSpinBox" name="m_TensorsToDWIBValueEdit">
           <property name="maximum">
            <number>10000</number>
           </property>
           <property name="singleStep">
            <number>100</number>
           </property>
           <property name="value">
            <number>1000</number>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QCommandLinkButton" name="m_TensorsToDWIButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="toolTip">
         <string>Estimates the original diffusion weighted image based on a reconstructed tensor image.</string>
        </property>
        <property name="statusTip">
         <string/>
        </property>
        <property name="whatsThis">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>Estimate DWI based on Tensor Image</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="title">
      <string>Estimate Q-Ball Image from Tensors</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <widget class="QCommandLinkButton" name="m_TensorsToQbiButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="toolTip">
         <string comment="Calculate ODF value as tensor value in the according direction" extracomment="Calculate ODF value as tensor value in the according direction">Calculate ODF value as tensor value in the according direction</string>
        </property>
        <property name="statusTip">
         <string/>
        </property>
        <property name="whatsThis">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>Start QBI Estimation</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_5">
     <property name="title">
      <string>Estimate Residuals</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="2" column="0">
       <widget class="QCheckBox" name="m_PercentagesOfOutliers">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>percentages of error</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QTabWidget" name="m_ResidualTab">
        <property name="currentIndex">
         <number>1</number>
        </property>
        <widget class="QWidget" name="tab">
         <attribute name="title">
          <string>Per volume</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_8">
          <item>
           <widget class="QmitkResidualAnalysisWidget" name="m_ResidualAnalysis" native="true">
            <property name="minimumSize">
             <size>
              <width>200</width>
              <height>300</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab_2">
         <attribute name="title">
          <string>Per slice</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_10">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>outliers per slice</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="frame_5">
            <property name="frameShape">
             <enum>QFrame::NoFrame</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QmitkResidualViewWidget" name="m_PerSliceView">
               <property name="maximumSize">
                <size>
                 <width>300</width>
                 <height>400</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QFrame" name="frame_4">
               <property name="frameShape">
                <enum>QFrame::NoFrame</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Raised</enum>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_9">
                <property name="leftMargin">
                 <number>0</number>
                </property>
                <property name="topMargin">
                 <number>0</number>
                </property>
                <property name="rightMargin">
                 <number>0</number>
                </property>
                <property name="bottomMargin">
                 <number>0</number>
                </property>
                <item>
                 <widget class="QLabel" name="m_UpperLabel">
                  <property name="text">
                   <string/>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QGraphicsView" name="m_LegendView">
                  <property name="maximumSize">
                   <size>
                    <width>20</width>
                    <height>255</height>
                   </size>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="m_LowerLabel">
                  <property name="text">
                   <string/>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="m_PositionLabel">
            <property name="text">
             <string>Volume: .., Slice:..</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QCommandLinkButton" name="m_ResidualButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="toolTip">
         <string>Calculate the residual from a dti and a dwi image</string>
        </property>
        <property name="statusTip">
         <string/>
        </property>
        <property name="whatsThis">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>Residual Image Calculation</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QmitkResidualAnalysisWidget</class>
   <extends>QWidget</extends>
   <header>QmitkResidualAnalysisWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QmitkResidualViewWidget</class>
   <extends>QGraphicsView</extends>
   <header>QmitkResidualViewWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
